contracts/cw1-subkeys/src/contract.rs

```diff
@@ -344,6 +348,42 @@ mod tests {
         }
     }
 
+        
+    #[test]
+    fn get_contract_version_works() {
```

Comment:
Thanks for the testcase. This could be a bit shorter (simple init), but this works as well.